lib/matplotlib/tests/test_contour.py

```diff
@@ -715,3 +715,10 @@ def test_bool_autolevel():
     assert plt.tricontour(x, y, z).levels.tolist() == [.5]
     assert plt.tricontourf(x, y, z.tolist()).levels.tolist() == [0, .5, 1]
     assert plt.tricontourf(x, y, z).levels.tolist() == [0, .5, 1]
+
+
+def test_all_nan():
+    x = np.array([[np.nan, np.nan], [np.nan, np.nan]])
+    assert_array_almost_equal(plt.contour(x).levels.tolist(),
```

Comment:
I'm pretty sure numpy's testing will compare the list and levels array for you, so I don't think you need to convert it to a list here unless you were doing the bare assert like above.
```suggestion
    assert_array_almost_equal(plt.contour(x).levels,
```

Reply:
Yeah right, removed `to_list` conversion. Thanks.